src/components/FullSizeText.vue

```diff
@@ -12,32 +12,30 @@
   </div>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts"> 
 import VueMarkdown from 'vue-markdown-render';
 import ModalLightbox from './ModalLightbox.vue';
+const { $requireMd } = useNuxtApp()
 
-export default defineComponent({
-  name: 'FullSizeText',
-  components: {
-    VueMarkdown,
-    ModalLightbox,
+const props = defineProps({
+  heading: {
+    required: false,
+    type: String,
   },
-  data() {
-    return {
-      text: require(`../assets/markdown/${this.textSource}`).default,
-    };
-  },
-  props: {
-    heading: {
-      required: false,
-      type: String,
-    },
-    textSource: {
-      required: true,
-      type: String,
-    },
+  textSource: {
+    required: true,
+    type: String,
   },
+})
+
+const md = $requireMd(props.textSource)
```

Comment:
Why not using `markdown` instead of cryptic `md`? Please use readable constant names.